test/uk/gov/hmrc/helptosave/connectors/ApiTwentyFiveCConnectorSpec.scala

```diff
@@ -18,35 +18,29 @@ package uk.gov.hmrc.helptosave.connectors
 
 import org.joda.time.LocalDate
 import org.scalatest._
-import org.scalamock.scalatest.MockFactory
 import play.api.libs.json.Json
 import uk.gov.hmrc.helptosave.config.WSHttp
 import uk.gov.hmrc.helptosave.models.{ApiTwentyFiveCValues, AwAwardStatus, Award}
 import uk.gov.hmrc.helptosave.util.NINO
+import uk.gov.hmrc.helptosave.utils.TestSupport
 import uk.gov.hmrc.play.http.{HeaderCarrier, HttpResponse}
 import uk.gov.hmrc.play.test.WithFakeApplication
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.concurrent.duration._
 
-class ApiTwentyFiveCConnectorSpec extends WordSpec with WithFakeApplication with Matchers with MockFactory {
+class ApiTwentyFiveCConnectorSpec extends WordSpec with WithFakeApplication with Matchers with TestSupport {
```

Comment:
you can make `TestSupport extends WordSpec with WithFakeApplication with Matchers ` so that you can get rid of these in each spec